openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -247,6 +250,40 @@ def validate_social_links(self, value):
             raise serializers.ValidationError("The social_links field must consist of unique social platforms.")
         return value
 
+    def validate_date_of_birth(self, value):
+        """
+        Method which validates date of birth that should be in the past.
+        """
+
+        value = self.convert_empty_to_None(value)
+
+        if value is not None and value > date.today():
```

Comment:
```
if value and value > date.today():
```
?

Reply:
Value can be date or None, so we check it here.